### PR TITLE
Changes taking Cloud Platform into account

### PR DESCRIPTION
**Pull request by MichalCz**

### CODE REVIEW

**Comment by MichalCz** [comment]
@a-tylenda please check.

File: README.md [hunk 1]
```diff
@@ -39,101 +34,95 @@
...
-<summary>
-    <strong>Host</strong>
-</summary>
+#### Log in to Scramjet Cloud Platform Beta Panel
 
-**Host** is responsible for maintaining and deploying Sequences, keeping them running and managing its lifecycle.
+<div className="w-200"></div>
 
-Host exposes also its own REST API to provide and receive data and manage Sequences and host itself.
+| ![Scramjet Welcome Page](images/welcome-page.png) |  To use our platform **free of charge for a year**, please [sign up for the beta program](https://scr.je/join-beta-docs). <br/> If you are already our user, please log in to the [Scramjet Cloud Platform Beta Panel](https://console.beta.scramjet.cloud/) through your provided mail and set the new password by clicking `Forgot the password`. |
```
Comment (a-tylenda) [inline README.md:63]:
```suggestion
| ![Scramjet Welcome Page](images/cp-welcome-page.png) |  To use our platform **free of charge for a year**, please [sign up for the beta program](https://scr.je/join-beta-docs). <br/> If you are already our user, please log in to the [Scramjet Cloud Platform Beta Panel](https://console.beta.scramjet.cloud/) through your provided mail and set the new password by clicking `Forgot the password`. |
```
I corrected the `.png` file name, but still, this file does not work, @MichalCz you need to upload it again
image

File: README.md [hunk 2]
```diff
@@ -39,101 +34,95 @@
...
+   ```
 
-</details>
+2. Generate SCP CLI Token.
 
-<details>
-<summary>
-    <strong>Host</strong>
-</summary>
+   [![Token will not be saved. After generation, please store it on your device.](../../images/screens/generate-token.png)](https://console.beta.scramjet.cloud/settings)
```
Comment (a-tylenda) [inline README.md:89]: File `generate-token.png` is missing.

File: README.md [hunk 3]
```diff
@@ -39,101 +34,95 @@
...
-* **Stream** - output can be streamed to a particular destination
-* you can mix multiple actions together: you can both send data to remote system/URL and save it locally.
+   ```bash
+   si seq start <sequence-id>
+   ```
 
-</details><br>
+5. After the program starts the Instance id will be returned.
+
+   Show output data form the running program (instance).
```
Comment (a-tylenda) [inline README.md:111]:
```suggestion
   Show output data from the running program (instance).
```

File: README.md [hunk 4]
```diff
@@ -27,10 +22,10 @@
 
 ## **Table of Contents**
 
-1. [Introduction](#1-introduction)
-    * [What is Scramjet Transform Hub](#11-what-is-scramjet-transform-hub-STH-?)
-    * [Solution concept diagram](#12-solution-concept-diagram)
-2. [Installation](#2-Installation)
+1. [Scramjet Cloud Platform](#1-scramjet-cloud-platform)
+    * [Introduction](#1-1-introduction)
+    * [Platform Quick Start](#1-2-platform-quick-start)
```
Comment (a-tylenda) [inline README.md:27]:
```suggestion
    * [Introduction](#11-introduction)
    * [Platform Quick Start](#12-platform-quick-start)
```
Section links should work now.

**Review by MichalCz — COMMENTED** [review]
Fix readme image location

**Pull request closed without merging**